Source/Core/Core/HW/EXI/BBA/BuiltIn.cpp

```diff
@@ -1,9 +1,14 @@
 // Copyright 2022 Dolphin Emulator Project
 // SPDX-License-Identifier: GPL-2.0-or-later
 
+#include <SFML/Network/Packet.hpp>
```

Comment:
Probably should be `#include "SFML/Network/Packet.hpp"` down by the other SFML includes.